helpers_core.go

```diff
@@ -32,5 +32,11 @@ func SetCreateVnicOptions(rawCreateVnicDetails interface{}) (vnicOpts *baremetal
 		*vnicOpts.AssignPublicIp = assignPublicIp.(string) == "1"
 	}
 
+	skipSourceDestCheck := vnic["skip_source_dest_check"]
+	if skipSourceDestCheck != nil {
+		vnicOpts.SkipSourceDestCheck = new(bool)
+		*vnicOpts.SkipSourceDestCheck = skipSourceDestCheck.(string) == "1"
```

Comment:
Does this always come through as "1" or is it ever possibly "true"?

Reply:
Good question... this is, as mentioned, a workaround for https://github.com/hashicorp/terraform/issues/13512. However, we're not using this approach everywhere where we have bools. Let me do a bit of testing around this and get back to you.